dashboard/lib/policies/child_account.rb

```diff
@@ -79,17 +80,17 @@ def self.user_predates_policy?(user)
     parent_permission_required?(user) && user.created_at < STATE_POLICY[user.us_state][:start_date]
   end
 
+  def self.state_policy(user)
+    return unless user.us_state
```

Comment:
You should probably merge with the latest `staging` changes and make sure the country_code check is still there: https://github.com/code-dot-org/code-dot-org/blob/1a66d6262ab0cd5bc91f11045724a8e4757ebfa0/dashboard/lib/policies/child_account.rb#L120